src/geouned/GEOUNED/Decompose/Decom_one.py

```diff
@@ -102,7 +102,7 @@ def cut_full_cylinder(solid):
             tools = (pp[0].shape,)
 
         try:
-            comsolid = UF.split_bop(solid, tools, opt.splitTolerance)
+            comsolid = UF.split_bop(solid, tools, options.splitTolerance, options)
```

Comment:
Here ```options.splitTolerance``` is duplicated with passing ```options``` class.
To me both options are suitable : 
if ``` UF.split_bop``` only use the splitTolerance attribute write the line as 
```suggestion
            comsolid = UF.split_bop(solid, tools, options.splitTolerance)
```
or for a general case
```suggestion
            comsolid = UF.split_bop(solid, tools, options)
```